CHANGELOG.md

```diff
@@ -28,9 +28,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
-- Added functionality to `sat bootsys boot --stage cabinet-power` to
-  automatically recreate Kubernetes cronjobs which are not being scheduled due
-  to missing too many scheduled start times.
+- Added functionality to `sat bootsys boot --stage platform-services` to
```

Comment:
This entry looks good. However, the changelog doesn't currently mention anything about automatic re-creation of the hms-discovery cronjob in the cabinet-power stage if it fails to be scheduled, which might be worth mentioning. It's related to the first item under Changed, but really might be its own separate item.